### PR TITLE
Create new indexer instances

### PR DESCRIPTION
**Pull request by gammazero**
New indexer nodes `alva`, `bria`, and `cora`. Each has 100Gi pvc size and are a mirror reader and writer.

### CODE REVIEW

File: deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/kustomization.yaml [hunk 1]
```diff
@@ -9,3 +9,6 @@ resources:
   - ber
   - cali
   - ago
+  - alva
+  - bria
+  - cora
```
Comment (masih) [inline deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/kustomization.yaml:14]: Postpone running them until they are configured to talk to a dhstore instance backed by FDB?
```suggestion

```

Comment (gammazero) [reply deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/kustomization.yaml]: They are all configured to use an assigner, but are not in the assigner's pool so will not be assigned anything. This means that they will not act on announcements received. So, running is OK and they will do nothing until the assigner is configured to assign to them.

We can certainly postpone running them until everything is ready. So I have changed to config as per above.

File: deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/alva/pvc_data.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    app.kubernetes.io/managed-by: kustomization
+    app.kubernetes.io/part-of: storetheindex
+  name: data
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 100Gi
+  storageClassName: io2
```
[the remark this answers is not in the document]
Comment (gammazero) [reply deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/alva/pvc_data.yaml]: done

**Review by masih — APPROVED** [review]

File: deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/kustomization.yaml (hunk 1 above)
Comment (masih) [reply deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/kustomization.yaml]: ty

**Pull request merged**